src/main/java/sg/ncl/MainController.java

```diff
@@ -2389,46 +2391,63 @@ public String saveExperimentImage(
             @PathVariable String expId,
             @PathVariable String nodeId) throws IOException {
 
-        if (saveImageForm.getImageName().length() < 2) {
+        String REDIRECT_SAVING_IMAGE = "redirect:/experiments/save_image/";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)